sui-execution/v0/sui-adapter/src/execution_engine.rs

```diff
@@ -449,6 +449,9 @@ mod checked {
             TransactionKind::AuthenticatorStateUpdate(_) => {
                 panic!("AuthenticatorStateUpdate should not exist in suivm");
             }
+            TransactionKind::RandomnessStateUpdate(_) => {
+                panic!("RandomnessStateUpdate should not exist in suivm");
```

Comment:
this panic message is a bit wrong (it's just as wrong for all the panics), it should say something like:

```suggestion
                panic!("RandomnessStateUpdate should not exist in execution layer v0");
```

Reply:
updated them